terms/ngs/commercialSource.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "sage.annotations-ngs.commercialSource-0.0.1",
+    "description": "Nucleic or amino acids characterized that originate from a commercial source.",
```

Comment:
Would it make sense to expand this term to use for commercial cell lines or stem cells? We've previously used individualIdSource to capture the vendor name for commercial cells, but this is definitely more specific.

Reply:
Yes, we can expand for commercial cell lines OR capture commercial products in individualIdSource. I thought individiualIdSource referred primarily to brain banks but I now realize if the dna/tissue/rna/cell line was of commercial source, there would not be redundancy with that key.

Reply:
I would say we can leave as is for now and expand the definition to use for cell products later if we need it. Thanks for confirming!